src/Core/MailTemplates/Handlebars/TrialInitiation.text.hbs

```diff
@@ -1,39 +1,50 @@
 {{#>FullTextLayout}}
-    Welcome to Bitwarden! You can now get started with secure credential management and extend the benefits of end-to-end encryption across your entire organization.
+Welcome to Bitwarden!
 
-    Your Master Password is the only way to unlock your account and only you hold the key. Memorize it, or write it down and keep it in a safe place.
+Here are a few simple steps to get up and running with Bitwarden Password Manager:
 
 
-    Install Bitwarden
-    ============
+1. Install the browser extension
+============
 
-    Access your Bitwarden account from anywhere and any device at the web vault ({{WebVaultUrl}}/?utm_source=welcome_email&utm_medium=email). For added convenience, download and install Bitwarden on any desktop, device and browser (http://www.bitwarden.com/download).
+Autofill passwords, save new logins, access the password generator, and more from the Bitwarden browser extension. The extension keeps Bitwarden easily accessible so you can be secure while online.
+Install the extension (http://www.bitwarden.com/download)
 
 
-    Download Options
-    ============
+2. Add passwords to your vault
+============
 
-    http://www.bitwarden.com/download
+Your secure vault is ready to safely store your sensitive information. Add logins, credit cards, notes, and identities for fast access and autofilling!
+Add passwords to your vault ({{WebVaultUrl}}/?utm_source=welcome_email&utm_medium=email)
```

Comment:
```suggestion
Add passwords to your vault ({{{WebVaultUrl}}}/?utm_source=welcome_email&utm_medium=email)
```

:pick: 
I would also triple escape in the `text.hbs` templates to stay consistent with the `html.hbs` files.

There are a few occurrences of this.

I did notice it was like this in the old version of these `text.hbs` templates, so up to you.
 

Triple stash `{{{`tells handlebars not to HTML-escape the input.

https://handlebarsjs.com/guide/expressions.html#html-escaping

Typically, we use `{{` for everything and only use triple for generated URLs, so everything is properly HTML escaped.

Reply:
@Thomas-Avery Thanks! I updated it to `{{{`